tests/test_db_service.py

```diff
@@ -95,13 +99,45 @@ async def test_replace_replaces_data(self):
     async def test_delete_deletes_data(self):
         """Test that delete method works and returns success."""
         self.collection.delete_one.return_value = futurized(
-            DeleteResult({}, True)
+            DeleteResult({'n': 1}, True)
         )
         success = await self.test_service.delete("test", self.id_stub)
         self.collection.delete_one.assert_called_once_with({"accessionId":
                                                            self.id_stub})
         self.assertTrue(success)
 
+    async def test_update_raises(self):
```

Comment:
following tests should be moved if raising is moved to operators

Reply:
refactored and done.